packages/amplify-graphql-searchable-transformer/.npmignore

```diff
@@ -0,0 +1,6 @@
+**/__mocks__/**
+**/__tests__/**
+src
+streaming-lambda
```

Comment:
ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.